app/javascript/src/hyacinth_v1/components/digital_objects/metadata/Field.js

```diff
@@ -10,17 +10,21 @@ import DateInput from '../../shared/forms/inputs/DateInput';
 import NumberInput from '../../shared/forms/inputs/NumberInput';
 import Checkbox from '../../shared/forms/inputs/Checkbox';
 
+const randomId = (prefix) => `${prefix}${Math.random()}`.replace('.','');
+
 const Field = (props) => {
   const {
-    onChange, value, dynamicField, dynamicField: { displayLabel, fieldType }
+    inputName, onChange, value, dynamicField, dynamicField: { displayLabel, fieldType }
   } = props;
 
   let field = '';
 
-  const sharedProps = { onChange, value };
+  const sharedProps = { onChange, value, inputName };
+  sharedProps.inputName ||= randomId('input-');
```

Comment:
```suggestion
  sharedProps.inputName = sharedProps.inputName || `field-${uniqueCheckboxIdCounter++}`;
```
Two suggestions here:
1) Guaranteed unique id instead of a random one.
2) I didn't think `||=` works in js.  Was it working in your local copy?

Reply:
Here's what threw me off:

```
let something ||= 'hello';
VM411:1 Uncaught SyntaxError: Unexpected token '||='
```
But:
```
let obj = {};
obj.something ||= 'hello';
"hello"
```

So I guess it only works for object properties.

Reply:
fixed!

Reply:
Done!